manta-benchmark/src/lib.rs

```diff
@@ -15,3 +15,94 @@
 // along with manta-rs.  If not, see <http://www.gnu.org/licenses/>.
 
 pub mod payment;
```

Comment:
Move this to underneath the other imports. See `CONTRIBUTING.md` file for more on this.